Framework/Algorithms/src/PolarizationCorrections/DepolarizedAnalyserTransmission.cpp

```diff
@@ -0,0 +1,215 @@
+       << DEPOL_OPACITY_NAME << "*x)";
+  return func;
+}
+} // namespace FitValues
+} // namespace
+
+namespace Mantid::Algorithms {
+
+using namespace API;
+using namespace Kernel;
+
+// Register the algorithm in the AlgorithmFactory
+DECLARE_ALGORITHM(DepolarizedAnalyserTransmission)
+
+std::string const DepolarizedAnalyserTransmission::summary() const {
+  return "Calculate the transmission rate through a depolarised He3 cell.";
+}
+
+void DepolarizedAnalyserTransmission::init() {
+  auto wsValidator = std::make_shared<WorkspaceUnitValidator>("Wavelength");
+  declareProperty(
+      std::make_unique<WorkspaceProperty<MatrixWorkspace>>(std::string(PropNames::DEP_WORKSPACE), "",
+                                                           Kernel::Direction::Input, wsValidator),
+      "The fully depolarised helium cell workspace. Should contain a single spectra. Units must be in wavelength.");
+  declareProperty(std::make_unique<WorkspaceProperty<MatrixWorkspace>>(std::string(PropNames::MT_WORKSPACE), "",
+                                                                       Kernel::Direction::Input, wsValidator),
+                  "The empty cell workspace. Must contain a single spectra. Units must be in wavelength");
+  declareProperty(std::string(PropNames::EMPTY_CELL_TRANS_START), FitValues::EMPTY_CELL_TRANS_START,
+                  "Starting value for the empty analyser cell transmission fit property " +
+                      std::string(FitValues::EMPTY_CELL_TRANS_NAME) + ".");
+  declareProperty(std::string(PropNames::DEPOL_OPACITY_START), FitValues::DEPOL_OPACITY_START,
+                  "Starting value for the depolarised cell transmission fit property " +
+                      std::string(FitValues::DEPOL_OPACITY_NAME) + ".");
+  declareProperty(std::string(PropNames::FIT_QUALITY_OVERRIDE), false,
+                  "Whether the algorithm should ignore a chi-squared (fit cost value) greater than 1 and therefore not "
+                  "throw an error.");
+  declareProperty(std::make_unique<WorkspaceProperty<ITableWorkspace>>(std::string(PropNames::OUTPUT_WORKSPACE), "",
+                                                                       Kernel::Direction::Output),
+                  "The name of the table workspace containing the fit parameter results.");
+  declareProperty(std::make_unique<WorkspaceProperty<MatrixWorkspace>>(
+                      std::string(PropNames::OUTPUT_FIT), "", Kernel::Direction::Output, PropertyMode::Optional),
+                  "The name of the workspace containing the calculated fit curve.");
+  declareProperty(std::make_unique<WorkspaceProperty<ITableWorkspace>>(
+                      std::string(PropNames::OUTPUT_COV_MATRIX), "", Kernel::Direction::Output, PropertyMode::Optional),
+                  "The name of the table workspace containing the normalised covariance matrix from the fit.");
```

Comment:
Sorry for getting confused, but is this outputting the normalised covariance matrix, or the non-normalised matrix?

Reply:
This is the non-normalised one. So good spot on the doc being wrong!